[DE] Add `Article_styling_criteria`, `Article_styling_criteria\Formatting` and `Article_styling_criteria\Writing` (#6451)
ppy/osu-wiki: wiki/Article_styling_criteria/Formatting/de.md

# Formatierung

> *Für die Schriftstandards, siehe: [Artikelgetsaltungskriterien/Schrift](../Writing)*

Reviewer: ```suggestion
*Für die Schriftstandards, siehe: [Artikelgestaltungskriterien/Schrift](../Writing)*
```

> *Hinweis: Dieser Artikel benutzt [RFC 2119](https://tools.ietf.org/html/rfc2119), um die Anforderungen zu beschreiben.*

Reviewer 2: this is not applicable if you are not using exact English words mentioned in the document -- same goes for `Writing` and `Article styling criteria` itself

Reviewer 2: on the other hand, I can remove this myself to avoid hindering the PR..

Es gibt einige Ausnahmen, in denen sich der Inhalt unterscheiden darf:

- Artikel, die ursprünglich in einer anderen Sprache als Englisch geschrieben wurden (in diesem Fall sollte Englisch als Übersetzung dienen)
- Erklärungen von englischen Begriffen, die gebräuchlich in der osu!-Community sind
- Externe Links
- Tags
- Subcommunity-spezifische Erklärungen

## Titelsektion

Metainformationen müssen sich ganz am Anfang der Datei befinden. Sie sind in [YAML](https://de.wikipedia.org/wiki/YAML#Beispiele) geschrieben und beschreiben zusätzliche Informationen über den Artikel. Diese müssen mit drei Bindestriche (`---`) auf der Zeile darüber sowie darunter umrandet werden. Ebenfalls muss eine leere Zeile zwischen dem YAML-Abschnitt und dem Titel vorhanden sein.

### Artikel, die Hilfe benötigen

*Tipp: Es wird empfohlen, englische Artikel mit diesem Tag nicht zu übersetzen. Ebenso sollte dieses Tag hinzugefügt werden, wenn die Übersetzung selbst aufgeräumt werden muss.*

> Das `needs_cleanup`-Tag darf bei Artikeln, die überarbeitet werden müssen oder Formatierungshilfe benötigen, hinzugefügt werden. Es ist auch akzeptabel ein Issue auf GitHub für diesen Zweck zu öffnen. Benutze dieses Tag wie folgt:

Author: yeah seems fine. If you turn the sentence around, you need to have a comma there: `Ein Issue zu öffnen, ist auch akzeptabel`.

```yaml
needs_cleanup: true
```

Sofern dieses Tag zu einem Artikel hinzugefügt wird, sollten [Kommentare](#kommentare) eingefügt werden, die erklären, was bei diesem Artikel überarbeitet werden muss.

### Nicht mehr aktuelle Artikel

*Tipp: Es wird empfohlen, englische Artikel mit diesem Tag nicht zu übersetzen. Wenn der englische Artikel diesen Tag trägt, dann muss die Übersetzung das ebenfalls.*

Übersetzte Artikel, die nicht mehr aktuell sind, müssen das `outdated`-Tag tragen, sobald die englische Variante aktualisiert wird. Englische Artikel dürfen auch mit "nicht mehr aktuell" gekennzeichnet werden, falls der Inhalt falsch, nicht eindeutig oder nicht mehr relevant ist. Das Tag muss folgendermaßen eingefügt werden:

```yaml
outdated: true
```

Sofern dieses Tag zu einem Artikel hinzugefügt wird, sollten [Kommentare](#kommentare) eingefügt werden, die erklären, was bei diesem Artikel überarbeitet werden muss.

### Artikel mit Tags versehen

> Tags helfen der Suchmaschine der Webseite relevantere Artikel anzuzeigen. Tags sollten in derselben Sprache, wie der Artikel geschrieben sein und zusätzlich noch die originalen Tags. Tags sollten nach Möglichkeit immer Kleinschreibung verwenden.

Reviewer: ```suggestion
Tags helfen der Suchmaschine der Webseite relevantere Artikel anzuzeigen. Tags sollten in derselben Sprache, wie der Artikel geschrieben sein und zusätzlich noch die originalen Tags. Außerdem sollten sie nach Möglichkeit immer Kleinschreibung verwenden.
```
LanguageTool complained about 3 sentences starting with the same word, so I switched it up.

> Ein Artikel über "Beatmap-Diskussion" könnte beispielweise diese Tags enthalten:

Reviewer: ```suggestion
Ein Artikel über "Beatmap-Diskussion" könnte beispielsweise diese Tags enthalten:
```

Artikeltitel sollten dem Ordnernamen entsprechen, in welchem sie sich befinden (Unterstriche (`_`) dürfen gerne an angemessenen Stellen mit Leerzeichen ersetzt werden). Wenn der Ordnername sich ändert, dann sollte der Titel sich ebenfalls ändern und umgekehrt.

---

Wettbewerbsartikel sind eine Ausnahme. Der Ordner muss Abkürzungen, Akronyme oder Initialien verwenden. Der Titel des Artikels muss den vollen Namen des Wettbewerbs enthalten.

## Ordner- und Dateistrukturen

### Ordnernamen

*Siehe auch: [Artikelbenennung](#artikelbenennung)*

Ordnernamen müssen in Englisch sein und "Sentence case" verwenden.

Ordnernamen dürfen nur diese Zeichen enthalten:

- Groß- und Kleinbuchstaben
- Zahlen
- Unterstriche (`_`)
- Bindestriche (`-`)
- Ausrufezeichen (`!`)

### Dateinamen für Artikel

Der Dateiname für einen Artikel kann in der Spalte `Dateiname` der Tabelle in [Locales](#locales) gefunden werden. Der übersetzte Artikel muss in denselben Ordner wie der englische Artikel abgelegt werden.

### Indexartikel

> Ein Indexartikel muss erstellt werden, wenn der Ordner nur dazu gedacht ist, andere Artikel zu beinhalten. Indexartikel müssen eine Liste an den enthaltenen Artikeln haben. Sie können auch weitere Informationen enthalten, z. B. einen Hauptabsatz oder eine Beschreibung der verlinkten Artikel.

Reviewer: ```suggestion
Ein Indexartikel muss erstellt werden, wenn der Ordner nur dazu gedacht ist, andere Artikel zu beinhalten. Indexartikel müssen eine Liste an den enthaltenen Artikeln haben. Sie können auch weitere Informationen enthalten, z. B. einen Hauptabsatz oder eine Beschreibung der verlinkten Artikel.
```
Amazing, this is a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space), which prevents an automatic line break. Yeah, also LanguageTool's idea, not mine xD

Author: The world of formatting and special characters is wonderful.

### Artikel für die Begriffsabgrenzung

[Begriffsabgrenzungsartikel](/wiki/Disambiguation) müssen in den Ordner `/wiki/Disambiguation` gesetzt werden. Die Hauptseite muss mit den neuen Artikeln aktualisiert werden. Siehe [Begriffsabgrenzung/Mod](/wiki/Disambiguation/Mod) als ein Beispiel.

Verlinkungen müssen aktualisiert werden, sodass mehrdeutige Begriffe zu den entsprechenden Begriffsabgrenzungsseiten zeigen.

Artikel, die aus einem Begriffsabgrenzungsartikel heraus verlinkt werden, müssen die Kopfnote ["Für andere Bedeutungen"](#für-andere-bedeutungen) tragen.

## HTML

HTML darf nicht verwenden werden, mit der Ausnahme von [Kommentaren](#kommentare). Die Struktur des Artikels muss überarbeitet werden, falls HTML verwendet wurde.

### Kommentare

> HTML-Kommentare sollten benutzt werden, um Todos hervorzuheben, aber auch, um Text zu kommentieren. Es wird empfohlen, sie auf eine eigene Zeile zu platzieren, aber es ist auch in Ordnung, wenn sie sich in einem Paragraphen befinden. Wenn das der Fall ist, darf der Start des Kommentars kein Leerzeichen haben.

Reviewer: ```suggestion
HTML-Kommentare sollten benutzt werden, um To-dos hervorzuheben, aber auch, um Text zu kommentieren. Es wird empfohlen, sie auf eine eigene Zeile zu platzieren, aber es ist auch in Ordnung, wenn sie sich in einem Paragraphen befinden. Wenn das der Fall ist, darf der Start des Kommentars kein Leerzeichen haben.
```
[Apparently, this is the correct spelling](https://www.duden.de/rechtschreibung/To_do). However, `Todos` is closer to the HTML comment `TODO`, so I would also be fine with keeping it as is. Same for the next two suggestions.

Schlechtes Beispiel:

```markdown

> HTML-Kommentare <!-- TODO explain HTML comments --> sollten benutzt werden, um Todos hervorzuheben, aber auch, um Text zu kommentieren.

Reviewer: ```suggestion
HTML-Kommentare <!-- TODO explain HTML comments --> sollten benutzt werden, um To-dos hervorzuheben, aber auch, um Text zu kommentieren.
```

```

Gutes Beispiel:

```markdown

> HTML-Kommentare<!-- TODO explain HTML comments --> sollten benutzt werden, um Todos hervorzuheben, aber auch, um Text zu kommentieren.

Reviewer: ```suggestion
HTML-Kommentare<!-- TODO explain HTML comments --> sollten benutzt werden, um To-dos hervorzuheben, aber auch, um Text zu kommentieren.
```

```

## Bearbeitung

### Zeilenende-Sequenz

*Vorsicht: Markdown-Dateien mit `CRLF` (carriage return and line feed) via GitHub hochzuladen, wird dazu führen, dass diese Dateien `CRLF` benutzen. Um das zu verhindern, setze das Zeilenende auf `LF` (line feed) vor dem Hochladen.*

> Markdown files must be checked in using the `LF` end of line sequence.

Author: Maybe `Zeilenende-Sequenz` for consistency? 🤔

> Markdown files must be checked in using the `LF` end of line sequence.

Reviewer 3: yeah true

### Escaping

Die Markdown-Syntax sollte bei Bedarf escaped werden. Artikeltitel werden jedoch als reiner Text gelesen und dürfen daher nicht escaped werden.

### Paragraphen

Jeder Paragraph muss von einer leeren Zeile gefolgt werden.

### Zeilenumbrüche

Zeilenumbrüche müssen ein Backslash (`\`) benutzen.

Zeilenumbrüche sollten nur selten verwendet werden.

## Kopfnote

*Nicht zu verwechseln mit [Hinweis](#hinweis).*

Kopfnoten sind kleine Notizen im oberen Teil eines Artikels oder eines Abschnitts, um Leser bei der Navigation zu verwandten Artikeln zu helfen oder sie über verwandte Artikel zu informieren.

Kopfnoten müssen in Kursivschrift verfasst sein und direkt unter der Überschrift liegen. Wenn mehrere Kopfnoten benutzt werden, dann müssen diese im selben Paragraphen sein; getrennt durch einen Zeilenumbruch.

### Hauptseite

> Die Kopfnote *Hauptseite* verweist den Leser zum Hauptartikel eines Themas. Wenn diese Kopfnote benutzt wird, wird impliziert, dass der Abschnitt eine Zusammenfassung des verlinkten Artikels ist. Die Kopfnote sollte nur einen Link enthalten. Sie muss wie folgt formartiert sein:

Reviewer: ```suggestion
Die Kopfnote *Hauptseite* verweist den Leser zum Hauptartikel eines Themas. Wenn diese Kopfnote benutzt wird, wird impliziert, dass der Abschnitt eine Zusammenfassung des verlinkten Artikels ist. Die Kopfnote sollte nur einen Link enthalten. Sie muss wie folgt formatiert sein:
```

> Die Knopfnote *Hauptseite* verweist den Leser zum Hauptartikel eines Themas. Wenn diese Knopfnote benutzt wird, wird impliziert, dass der Abschnitt eine Zusammenfassung des verlinkten Artikels ist. Die Knopfnote sollte nur einen Link enthalten. Sie muss wie folgt formartiert sein:

Author: I had to read this at least three times, before seeing the n that is too much xD No idea how that came in there. Maybe I was tired when I wrote this ...

```markdown
*Hauptseite: {Artikel}*

*Hauptseiten: {Artikel} und {Artikel}*
```

### Siehe auch

Kopfnoten mit *Siehe auch* empfehlen dem Leser andere interessante Artikel oder Abschnitte. Diese müssen wie folgt aussehen:

```markdown
*Siehe auch: {Artikel}*

*Siehe auch: {Artikel} und {Artikel}*
```

### Für siehe

> Kopfnoten *Für siehe* sind den *Siehe auch*-Kopfnoten ähnlich, aber im Allgemeinen beschreibender und direkter. Diese Kopfnoten dürfen mehrere Links enthalten, falls erfoderlich. Sie müssen wie folgt geschrieben werden:

Reviewer: ```suggestion
Kopfnoten *Für siehe* sind den *Siehe auch*-Kopfnoten ähnlich, aber im Allgemeinen beschreibender und direkter. Diese Kopfnoten dürfen mehrere Links enthalten, falls erforderlich. Sie müssen wie folgt geschrieben werden:
```

Namen von Videospielen sollten in kursiv sein. osu!—das Spiel—ist davon ausgenommen.

Das erste Vorkommen einer Abkürzung, eines Akronyms oder eines Intials darf kursiv sein.

Kursivschrift darf auch benutzt werden, um die Lesbarkeit zu verbessern oder um Dinge hervorzuheben.

## Überschriften

Alle Überschriften müssen "sentence case" benutzen.

Überschriften müssen den [ATX-Stil (hash)](https://github.github.com/gfm/#atx-headings) verwenden und müssen eine leere Zeile davor sowie danach haben. Die Titelüberschrift ist eine Ausnahme, sofern sie auf der ersten Zeile liegt. Sollte das der Fall sein, muss nur eine leere Zeile nach der Überschrift vorhanden sein.

Überschriften dürfen nicht das Überschriftenlevel 5 überschreiten und dürfen nicht genutzt werden, um Text zu formatieren.

### Titel

*Siehe auch: [Artikelbenennung](#artikelbenennung)*

*Vorsicht: Titel werden als Klartext verarbeitet, sie dürfen nicht escaped werden.*

Die erste Überschrift muss in allen Artikeln das Level 1 haben, da es der Artikeltitel ist. Alle Überschriften danach müssen [Abschnittstitel](#abschnitte) sein. Titel dürfen keine Formatierung, Links oder Bilder enthalten.

> Die Titelüberschrift muss sich auf der erten Zeile befinden, außer es werden [Metainformationen](#titelsektion) genutzt. Sollte das der Fall sein, muss der Titel sich darunter befinden; mit einer leeren Zeile dazwischen.

Author: yeah, I can already see it coming. Opening a PR because one word in the article has a typo xD

```

### Nummerierte Listen

Die Zahlen in einer nummerierten Liste müssen erhöht werden, um ihren Schritt darzustellen.

```markdown
1. Lade das Installationsprogramm für osu! herunter.
2. Führe das Installationsprogramm aus.
   1. Klicke auf den Text unter der Fortschrittsanzeige, um den Installationsort zu ändern.
   2. Das Installationsprogramm wird dich nach einem Ort fragen. Gib ein, wo du osu! installieren möchtest.
3. osu! wird gestartet, sobald die Installation fertig ist.
4. Logge dich ein.
```

### Gemischt

Aufzählungslisten gemeinsam mit nummerierten Listen sollte nur selten benutzt werden.

```markdown
1. Lade einen Skin aus den Foren herunter.
2. Lade die Skindatei in osu!.
   - Wenn die Datei eine `.zip` ist, entpacke sie erst und verschiebe den Inhalt in den `Skins/`-Ordner (in deinem osu!-Installationsordner).
   - Wenn die Datei eine `.osk` ist, öffne sie auf deinem Desktop oder verschiebe sie per Drag-and-drop in den Spiel-Client.
3. Öffne osu!, falls es nicht offen ist, und wähle einen Skin in den Optionen.
   - Das kann bereits passiert sein, falls du die `.osk`-Datei geöffnet oder sie per Drag-and-drop in den Spiel-Client verschoben hast.
```

## Code

> Das Mark-up für Code ist ein Akut (`` ` ``). Um Aktue in Code zu platzieren, nutze bitte doppelte Akute. Wenn der Akut am Anfang oder am Ende steht, füge ein Leerzeichen ein. (Beispiel unten gezeigt.)

Reviewer: ```suggestion
Das Mark-up für Code ist ein Akut (`` ` ``). Um Akute in Code zu platzieren, nutze bitte doppelte Akute. Wenn der Akut am Anfang oder am Ende steht, füge ein Leerzeichen ein. (Beispiel unten gezeigt.)
```

```markdown
Der `osu!direct`-Button ist im Hauptmenü auf der rechten Seite sichtbar, sofern du ein aktives osu!supporter-Tag hast.
```

### Ordner- und Verzeichnisnamen

Beim Kopieren des Namens eines Ordners oder eines Verzeichnisses muss die Schreibweise so sein, wie sie angezeigt wird. Es wird hier trotzdem die Kleinschreibung empfohlen, sofern möglich. Verzeichnispfade dürfen nicht absolut sein (also starte den Verzeichnisnamen nicht vom Laufwerksbuchstaben oder Wurzelpfad aus). (Beispiel unten gezeigt.)

```markdown
osu! ist standardmäßig im Ordner `AppData/Local` installiert, außer der Installationsort wurde anders festgelegt während der Installation.
```

### Schlüsselwörter und Befehle

Beim Kopieren von Schlüsselwörtern oder Befehlen muss die Schreibweise so sein, wie sie angezeigt wird oder wie sie jemand normalerweise schreiben würde. Sofern anwendbar, sollte Kleinschreibung verwendet werden. (Beispiele unten gezeigt.)

```markdown
Bis jetzt machen die Befehle `Name` und `Author` in der Skinkonfigurationsdatei (`skin.ini`) nichts.
```

### Dateinamen

> Beim Kopieren von Dateinamen sollte die Schreibweise so sein, wie sie angezeigt wird. Sofern andwendbar, wird Kleinschreibung empfohlen. (Beispiele unten gezeigt.)

Reviewer: ```suggestion
Beim Kopieren von Dateinamen sollte die Schreibweise so sein, wie sie angezeigt wird. Sofern anwendbar, wird Kleinschreibung empfohlen. (Beispiele unten gezeigt.)
```

### Interne Links

*Tipp: Mit internen Links sind Links gemeint, die in der Domäne `https://osu.ppy.sh/` liegen.*

#### Wiki-Links

Alle Links, die auf einen Wikiartikel zeigen, sollten mit `/wiki/` anfangen, gefolgt von dem Pfad zu dem Artikel. Relative Links dürfen auch benutzt werden. Einige Beispiele sind unten aufgelistet:

```markdown
[FAQ](/wiki/FAQ)
[pippi](/wiki/Mascots#pippi)
[Beatmaps](../)
[Pattern](./Pattern)
```

Wiki-Links dürfen keine Umleitungen benutzen und dürfen keinen Schrägstrich (`/`) am Ende haben.

Schlechte Beispiele:

```markdown
[Artikelgestaltungskriterien](/wiki/ASC)
[Entwickler](/wiki/Developers/)
[Entwickler](/wiki/Developers/#game-client-developers)
```

Gute Beispiele:

```markdown
[Artikelgestaltungskriterien](/wiki/Article_styling_criteria)
[Entwickler](/wiki/People/The_Team/Developers)
[Entwickler](/wiki/People/The_Team/Developers#game-client-developers)
```

##### Links zu Unterartikeln

Wiki-Links, die zu einem Unterartikel zeigen, sollten den Namen des übergeordneten Ordners in ihrem Linktext haben. Siehe das folgende Beispiel:

```markdown
*Siehe auch: [Beatmap-Editor/Design](/wiki/Beatmap_Editor/Design)*
```

##### Links zu Abschnitten

> *Hinweis: Den Überschriftenleveln 4 und 5 wird keine Kennung zugewiesen. D. h., dass nicht direkt auf sie verwiesen werden kann.*

Reviewer: ```suggestion
*Hinweis: Den Überschriftenleveln 4 und 5 wird keine Kennung zugewiesen. D. h., dass nicht direkt auf sie verwiesen werden kann.*
```
Another non-breaking space :)

Wenn zu anderen Benutzerprofilen verlinkt wird, muss die Benutzer-ID verwenden werden. Benutze die neue Webseite (`https://osu.ppy.sh/users/{username})`), um an die Benutzer-ID zu gelangen.

Der Linktext des Benutzerlinks sollte den aktuellen Namen des Benutzers verwenden.

##### Schwierigkeitsgrade

Sobald ein einziger Schwierigkeitsgrad verlinkt wird, sollte folgendes Format für den Linktext verwendet werden:

```
{Künstler} - {Titel} ({Ersteller}) [{Name_des_Schwierigkeitsgrads}]
```

Der Link muss tatsächlich auf den Schwierigkeitsgrad verweisen. URLs für Beatmap-Schwierigkeitsgrade müssen wie folgt formatiert sein:

```
https://osu.ppy.sh/beatmapsets/{BeatmapSetID}#{mode}/{BeatmapID}
```

Der Name des Schwierigkeitsgrads darf aus dem Linktext weggelassen werden, muss dann aber konsistent im ganzen Artikel sein.

##### Beatmaps

Sobald auf eine Beatmap verwiesen wird, benutze dieses Format für den Linktext:

```
{Künstler} - {Titel} ({Ersteller})
```

> Alle URLs zu Beatmaps müssn folgendermaßen formatiert sein:

Reviewer 3: ```suggestion
Alle URLs zu Beatmaps müssen folgendermaßen formatiert sein:
```

```
https://osu.ppy.sh/beatmapsets/{BeatmapSetID}
```

### Externe Links

*Hinweis: Mit externen Links sind Links gemeint, die außerhalb der Domäne `https://osu.ppy.sh/` liegen.*

> Das Protokoll `https` muss benutzt werden, es sei denn, die Seite unterstützt dies nicht. Externe Links müssen eine klaren und direkten Link zu eine seriöse Quelle sein. Die URL aus der Adressleiste deines Webbrowsers sollte so kopiert werden, wie sie bei der Verlinkung auf externe Seiten verwendet wird.

Reviewer 3: ```suggestion
Das Protokoll `https` muss benutzt werden, es sei denn, die Seite unterstützt dies nicht. Externe Links müssen eine klaren und direkten Link zu einer seriösen Quelle sein. Die URL aus der Adressleiste deines Webbrowsers sollte so kopiert werden, wie sie bei der Verlinkung auf externe Seiten verwendet wird.
```

### Anzeige

Wenn ein Bild der einzige Inhalt eines Absatzes ist, dann wird dieses als zentrierter Block angezeigt. Ansonsten liegt es dynamisch im umgebenden Inline-Text.

Blockbilder mit einem Titeltext zeigen den Titeltext als eine Bildunterschrift unter dem Bild an.

Blockbilder werden häufig mit einer Infobox<!-- TODO: link me! --> zusammen formatiert, um die anfängliche Größe zu reduzieren und damit diese sich auf der Seite zu anderem Inhalt befinden:

```markdown
::: Infobox
![](img/mod-response.png "Ein Beispiel einer Antwort eines Moderators")
:::
```

### Bild-Caching

Bilder werden bis zu 60 Tage im Cache gespeichert. Das Bild wird mit der URL des Bildlinks zugeordnet.

Wenn ein Bild aktualisiert wird, dann muss entweder der Bildname verändert oder ein Suchbegriff an die URL angehängt werden. In beiden Fällen müssen alle Übersetzungen mit dem neuen Bildlink versehen werden.

### Formate und Qualität

> Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80%, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.

Reviewer 3: ```suggestion
Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80%, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das Format GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.
```

Author: GIF (Graphics Interchange Format) already has `Format` in its name, so I thought that writing something like _das Format Graphics Interchange Format_ is wrong.

Reviewer 3: hmmm actually, i guess technically it would be correct without the `Format`, just sounded better to me xd

Author: another solution might be `GI-Format`? Though I'm not sure if everyone will understand this at a glance then because GIF is so widely used.

Reviewer 3: just GIF is good then

> Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80%, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.

Reviewer: ```suggestion
Bilder sollten das Format JPG bei einer Qualität von 8 (80 oder 80 %, abhängig vom Programm) verwenden. Wenn das Bild (teilweise) transparent ist oder lesbaren Text enthält, dann muss stattdessen das Format PNG benutzt werden. Wenn das Bild eine Animation enthält, kann das GIF eingesetzt werden; jedoch sollte dies nur sparsam verwendet werden, da diese länger zum Laden benötigen und auch größer als die [maximale Dateigröße](#dateigröße) sein können.
```
I'm (or, better, LT is) really nitpicking at this point. Hope you don't mind xD

Author: [TIL](https://www.texterclub.de/typische-rechtschreibfehler-leerzeichen/) that you put a space between a number and the percent character.

### Dateigröße

> Bilder müssen weniger als 1 MB groß sein, ansonsten werden sie nicht geladen. Eine Herunterskalierung und die Benutzung von JPG bei 80% reicht dafür meistens aus.

Reviewer: ```suggestion
Bilder müssen weniger als 1 MB groß sein, ansonsten werden sie nicht geladen. Eine Herunterskalierung und die Benutzung von JPG bei 80 % reicht dafür meistens aus.
```

> Bilder müssen unter 1 MB liegen, ansonsten werden sie nicht geladen. Eine Herunterskalierung und die Benutzung von JPG bei 80% reicht dafür meistens aus.

Author: Doesn't it need to be `weniger als`? Or is that colloquial form?

Alle Screenshots des Spielverlaufs müssen in der Updatequelle Stable gemacht werden, außer es ist ein spezifisches Feature, welches nicht in Stable vorhanden ist. Du solltest die Screenshot-Funktion des Spiels (`F12`) nutzen.

#### Einstellungen des Spiel-Clients

*Tipp: Wenn du deine aktuellen Einstellungen für das Wiki nicht ändern möchtest, dann kannst du die Datei `osu!.<ComputerUser>.cfg` aus deinem osu!-Ordner entfernen und später wieder hinzufügen.*

Du musst diese Einstellungen im Spiel-Client haben, bevor du einen Screenshot erstellst (Einstellungen, die unten nicht genannt werden, sollten auf ihrem Standardwert stehen):

- Sprache wählen: `Englisch`
- Metadaten in Originalsprache: `Aktiviert`
- Bildschirmauflösung: `1280x720`
- Vollbildmodus: `Deaktiviert`
- Parallaxeneffekt: `Deaktiviert`
- Menütipps: `Deaktiviert`
- Seasonale Hintergünde: `Niemals`
- Tasten-Overlay: `Aktiviert`
- Aktueller Skin: `Standard` (erste Option)

*Hinweis für Übersetzer: Wenn du einen Artikel mit Screenshots des Spiels übersetzt, dann darf die Sprache des Spiel-Clients in der übersetzten Sprache sein.*

### Bilderlinks

Bilder dürfen nicht Teil eines Linktextes sein.

> Flaggensymbole neben Links zu Benutzern müssen separat vom Linktext sein. Siehe folgendes Beispiel:

Author: `müssen` is there two times. This shouldn't be like this, should it?

```markdown
![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
```

### Flaggensymbole

*Für eine Liste der Flaggensymbole, siehe: [Issue \#328](https://github.com/ppy/osu-wiki/issues/328)*

> Die Flaggensymbole benutzen eine Zwei-Buchstaben-Kodierung (alle in Großschreibweise) und enden in `.gif`. Benutze beim Hinzufügen eines Symbols in der Linie dieses Format:

Reviewer: ```suggestion
Die Flaggensymbole benutzen eine Zwei-Buchstaben-Kodierung (alle in Großschreibweise) und enden in `.gif`. Benutze beim Inline-Hinzufügen eines Symbols dieses Format:
```
Might as well use `inline` here, too. Or maybe change `in der Linie` to `in der Zeile` or `im Text` even. I'm still not really happy with my suggestions either, but I can't think of anything better ^^

Author: I think Inline is fine for the time being. Sometimes, there isn't any good word for translation.

```markdown
![](/wiki/shared/flag/xx.gif)
```

Wobei `xx` der [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) zwei-Buchstaben Ländercode für die Flagge ist.

> Der volle Ländername sollte in den Titeltext eingefügt werden. Dr Ländercode im alternativen Text ist optional, aber muss bei allen Flaggen im Artikel angewendet werden.

Reviewer 3: ```suggestion
Der volle Ländername sollte in den Titeltext eingefügt werden. Der Ländercode im alternativen Text ist optional, aber muss bei allen Flaggen im Artikel angewendet werden.
```

## Tabellen

Tabellen auf der Webseite unterstützen Titel nur in der ersten Zeile.

Tabellen müssen nicht schön gemacht werden (füge keine extra Leerzeichen ein, um die Abstände gleichmäßig zu machen). Sie müssen einen senkrechten Strich (`|`) auf der linken sowie rechten Seite haben und jede Zelle muss mit einem Leerzeichen auf jeder Seite versehen sein. Leere Zellen müssen einen senkrechten Strich, gefolgt von zwei Leerzeichen, gefolgt von einem weiteren senkrechten Strich haben.

> Die Trennzeichen (die nächste Zeile nach den Spaltenüberschriften) darf nur drei Zeichen pro Spalte benutzen (und mit einem Leerzeichen auf jeder Seite versehen sein), die wie folgt aussehen müssen:

Reviewer 3: ```suggestion
Die Trennzeichen-Zeile (die nächste Zeile nach den Spaltenüberschriften) darf nur drei Zeichen pro Spalte benutzen (und mit einem Leerzeichen auf jeder Seite versehen sein), die wie folgt aussehen müssen:
```

> - `:--` (linksbüngig)

Reviewer: ```suggestion
- `:--` (linksbündig)
```

- `:-:` (mittig ausrichten)
- `--:` (rechtsbündig)

---

Das folgende ist ein Beispiel, wie eine Tabelle aussehen sollte:

```markdown
| Team "Picturesque" Rot | Punktestand | Team "Statuesque" Blau | Durchschnittliche Beatmap-Sterne |
| :-- | :-: | --: | :-- |
| **peppy** | 5 - 2 | pippi | 9.3 Sterne |
| Aiko | 1 - 6 | **Alisa** | 4.2 Sterne |
| Ryūta | 3 - 4 | **Yuzu** | 5.1 Sterne |
| **Taikonator** | 7 - 0 | Tama | 13.37 Sterne |
| Maria | Kein ausgetragenes Spiel | Mocha |  |
```

## Blockzitate

Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren.

## Thematische Brüche

> Thematische Brüche (auch bekannt als horizontale Linie) sollte nur selten verwendet werden. Ein paar sinnvolle Benutzungen sind hier aufgelistet (aber nicht darauf beschränkt):

Reviewer: ```suggestion
Thematische Trennungen (auch bekannt als horizontale Linien) sollten nur selten verwendet werden. Ein paar sinnvolle Benutzungen sind hier aufgelistet (aber nicht darauf beschränkt):
```

- Bilder von Text trennen
- Mehrere Bilder, die direkt hintereinander sind, voneinander trennen
- Das Thema innerhalb eines Abschnitts wechseln

> Diese müssen eine leere Zeile vor und nach dem Markup haben. Thematische Brüche dürfen nur drei Bindestriche nutzen, wie unten gezeigt:

Reviewer: ```suggestion
Diese müssen eine leere Zeile vor und nach dem Markup haben. Thematische Trennungen dürfen nur drei Bindestriche nutzen, wie unten gezeigt:
```
`Unterbrechung` or `Umbruch` might work too (also on line 772), but `Bruch` sounds wrong to me.
